.github/wiki/_Sidebar.md

```diff
@@ -1,6 +1,7 @@
 ## Table of Contents
 
 - [[Home]]
+- [[Generation]]
```

Comment:
Don't forget to update the link on the Contribute page @topnimble 

![image](https://user-images.githubusercontent.com/11751745/157819390-4f2851ee-ece3-4096-9c0e-5f0906d8328f.png)

Reply:
Fixed in d78d0f0